src/github/client/github-client.ts

```diff
@@ -226,12 +224,22 @@ export default class GitHubClient {
 	public async getBranchesPage(owner: string, repoName: string, perPage?: number, cursor?: string): Promise<GetBranchesResponse> {
 		const response = await this.graphql<GetBranchesResponse>(GetBranchesQuery,
 			{
-				owner: owner,
+				owner,
 				repo: repoName,
 				per_page: perPage,
 				cursor
 			});
 		return response?.data?.data;
 	}
 
+	public async getCommitsPage(includeChangedFiles: boolean, owner: string, repoName: string, perPage?: number, cursor?: string | number): Promise<getCommitsResponse> {
```

Comment:
I don't particularly agree with having `includeChangedFiles` at the start of the argument list.  The argument list is also starting to become unmanageable.  I suggest we use a single object type for all params.  `includeChangedFiles` should be options, defaulted to false.

Reply:
ha, agreed, I had originally created a repoDetails object for exactly that but stripped it out again when I dropped the default ref value. But yes args are getting too long. I'll do some tidy up.